src/site/content/en/handbook/adding-a-course/index.md

```diff
@@ -6,22 +6,96 @@ description: |
   Learn how to add a new course to the site.
 ---
 
-## Required files
+## Add your content
 
-To add a new course you will need to create the following files:
+Create a new directory in `src/site/content/en/learn/`. The name of this
+directory will be your `projectKey` which we'll use later. This will also be
+your courses' url. For example, the directory `src/site/content/en/css` has a
+`projectKey` of `css` and is accessible at [web.dev/learn/css](https://web.dev/learn/css).
 
-- `src/site/_data/i18n/courses.yml`
-- `src/site/_data/courses/forms/meta.yml`
-- `src/site/_data/courses/forms/toc.yml`
-- `src/site/content/en/learn/forms/example/index.md`
-- `src/site/content/en/learn/forms/forms.11tydata.js`
-- `src/site/content/en/learn/forms/index.md`
+Add an `index.md` to the root of this directory. This will be your courses'
+landing page. You can [copy the one from Learn CSS](https://github.com/GoogleChrome/web.dev/blob/main/src/site/content/en/learn/css/index.md) as a starter.
 
-You can refer to [the #5964 pull request](https://github.com/GoogleChrome/web.dev/pull/5964) as an example.
+Finally, add an `11tydata.js` file in this directory. The file name should match
+the directory name. So if your directory is named `css`, you will add a file
+named `css.11tydata.js`.
 
-### Course Layout
+Add this content to your `11tydata.js` file. **Be sure to update the
```

Comment:
This isn't feedback on this PR per se, but this is a lot of boilerplate we need to add. I might look to see if we can make this be created by a defaults helper function, since for most courses it'll be exactly the same.

Reply:
yeah i agree :P